database/sql/V034__create_index_order_owner.sql

```diff
@@ -0,0 +1,4 @@
+-- replace index for effective order searching
+DROP INDEX public.user_order_creation_timestamp;
+
+CREATE INDEX order_owner ON public.orders USING HASH (owner);
```

Comment:
```suggestion
DROP INDEX user_order_creation_timestamp;

CREATE INDEX order_owner ON orders USING HASH (owner);
```

Is there a reason we need `public` schema here? We don't use it in other migration scripts and would slightly prefer not to have it for consistency with our other SQL scripts.

Reply:
I didn't notice this while reviewing. Agree with the suggestion.